apps/teams-test-app/src/components/AppAPIs.tsx

```diff
@@ -98,7 +98,6 @@ const RegisterBeforeSuspendOrTerminateHandler = (): React.ReactElement =>
     title: 'Register Before Suspend/Terminate Handler',
     onClick: async (setResult) => {
       app.lifecycle.registerBeforeSuspendOrTerminateHandler((): void => {
-        alert('beforeSuspendOrTerminate received');
         setResult('Success');
```

Comment:
Does it make sense to change the setResult to 'beforeSuspendOrTerminate received'? instead of success?